tier4_external_api_msgs/srv/CalibrationDataRequest.srv

```diff
@@ -0,0 +1,5 @@
+bool data_request
+---
+uint8[] graph_image
+float32[] accel_map
+float32[] brake_map
```

Comment:
The format of accel_map and brake_map is a CSV, so we need a 2D array or column size.
Moreover, the first column may be the string "default".